app/src/main/java/org/hackillinois/android/common/FavoritesManager.kt

```diff
@@ -12,12 +13,22 @@ class FavoritesManager {
             HackIllinoisNotificationManager.scheduleEventNotification(context, it)
         }
 
+        fun favoriteProject(context: Context, project: Project?) = project?.let {
+            setBoolean(context, it.name, true)
```

Comment:
Can you use the id instead of the name here? if there isn't an id field in the Project class, you can add it and annotate it the @PrimaryKey instead of the name.